app/lib/features/home/widgets/my_spaces_section.dart

```diff
@@ -87,32 +82,28 @@ class _RenderSpacesSection extends ConsumerWidget {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.stretch,
       children: [
-        title,
+        Row(
+          children: [
+            title,
+            const Spacer(),
+            ActerInlineTextButton(
+              onPressed: () => context.pushNamed(Routes.spaces.name),
+              child: Text(L10n.of(context).seeAll),
+            ),
+          ],
+        ),
+        const SizedBox(height: 8),
         ListView.builder(
           shrinkWrap: true,
           itemCount: limit,
           physics: const NeverScrollableScrollPhysics(),
           itemBuilder: (context, index) {
-            return SpaceCard(space: spaces[index]);
+            return SpaceCard(
+              space: spaces[index],
+              margin: const EdgeInsets.only(bottom: 14),
+            );
           },
         ),
-        if (showAll)
```

Comment:
another removing of a feature (`showAll`) that wasn't reflected in the Widgets API updates... please remove `showAll` from the parameters to make sure this isn't breaking the assumptions somewhere.

Reply:
Let me put back (`showAll`) condition again for "see all" button. I was thinking to see all button irrespective of any conditions, but I think this condition is good and doesn't require see all button when we have only 2 space or something.